controller/test/blower_fsm/blower_fsm_test_data.h

```diff
@@ -0,0 +1,1061 @@
+/* Copyright 2020f,
+RespiraWorks
+Licensed under the Apache Licensef,
+Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writingf,
+software
+distributed under the License is distributed on an "AS IS" BASISf,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KINDf,
+either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#ifndef BLOWER_FSM_PRESSURE_ASSIST_TEST_DATA_H
+#define BLOWER_FSM_PRESSURE_ASSIST_TEST_DATA_H
+
+#include "units.h"
+
+// Trace of net flow readings.  Captured using pressure command mode at covent
+// preset #2:
+//
+//  - PIP  20cm
+//  - PEEP 10cm
+//  - rr   12bpm (5s/breath)
+//  - i:e  0.25 (itime=1s, etime=4s)
+//
+// Duration between each line is 10ms.
+//
+// There's a manual inspiratory attempt in here; in other words, I expanded the
+// test lung a bit during the expiratory dwell.
+inline const VolumetricFlow FLOW_TRACE_1[] = {
```

Comment:
Will this data ever need to be updated? If yes, how would one do it? Or basically, how did you capture the data and create this .h file?

Reply:
Expanded the comment a bit, although I'm not sure if this is enough.  LMK if there's anything else missing and I can go back and add it

```
// Trace of net flow readings captured on a live device using pressure command
// mode at covent preset #2:
//
//  - PIP  20cm
//  - PEEP 10cm
//  - rr   12bpm (5s/breath)
//  - i:e  0.25 (itime=1s, etime=4s)
//
// Duration between each line is 10ms.
//
// There's a manual inspiratory attempt in here; in other words, I expanded the
// test lung a bit with my finger during the expiratory dwell.
```